tests/std/tests/P0288R9_move_only_function/test.cpp

```diff
@@ -0,0 +1,565 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cstdlib>
+#include <functional>
+#include <stdexcept>
+#include <type_traits>
+#include <utility>
+#include <vcruntime_new.h>

```

Comment:
I think we should include Standard `<new>` for `bad_alloc` - unless I'm missing a reason why we need `<vcruntime_new.h>` specifically. (That is, we have a weak preference for writing Standard code in tests - makes things somewhat more maintainable, and might make applying our tests to libc++ easier in the future.)